### PR TITLE
CompressAs fix

### PR DESCRIPTION
**Pull request by anddrrw**
+ Fix one CompressAs issue with tempfile being in use while attempting to remove
+ Better GPS formatting for HP tool
+ More descriptive CSV names

### CODE REVIEW

**Comment by anddrrw** [comment]
This is a fix for a bug with CompressAs that I had with ex-20. I'm still researching what's going on with Emily's...maybe hold off on merging for a bit, it would be great to get that fix in here too.

File: plugins/CompressAs/__init__.py [hunk 1]
```diff
@@ -117,12 +117,13 @@ def save_as(source, target, qTables):
     im.save(target, subsampling=1, qtables=finalTable)
     if thumbTable:
         im.thumbnail((128,128))
-        tempFile = tempfile.mkstemp(suffix='.jpg')[1]
+        fd, tempFile = tempfile.mkstemp(suffix='.jpg')[1]
```
Comment (rwgdrummer) [inline plugins/CompressAs/__init__.py]: Need to remove the [1]

File: hp_tool/hp_data.py [hunk 2]
```diff
@@ -82,6 +82,20 @@ def parse_prefs(data):
 
     return newData
 
+def parse_GPS(coordinate):
```
Comment (rwgdrummer) [inline hp_tool/hp_data.py]: what is the intent here?
Can you use the latlon package?

Comment (anddrrw) [inline hp_tool/hp_data.py]: I'd rather not require another dependency for something simple like this...it just converts the specific exiftool output of lat/lon from DMS to decimal degrees.

**Pull request merged**